src/Query/Interpolator.php

```diff
@@ -141,4 +132,27 @@ private static function escapeStringValue(string $value): string
             '\\' => '\\\\',
         ]);
     }
+
+    /**
+     * Categorizes parameters into named and unnamed.
+     *
+     * @param iterable $parameters Parameters to categorize.
+     *
+     * @return array An associative array with keys 'named' and 'unnamed'.
```

Comment:
Probably psalm typings are needed here, to define that array has two keys: 'named', 'unnamed'. Or rollback this change to initial.